Makefile

```diff
@@ -90,6 +90,10 @@ sandbox-release: neard-sandbox-release
 neard-sandbox-release:
 	cargo build -p neard --features sandbox --release
 
+statelessnet-release: NEAR_RELEASE_BUILD=release
+statelessnet-release:
+	cargo build -p neard --release --features statelessnet_protocol
+
```

Comment:
I didn't add store validator and genesis update because I don't know if they are needed.